exercises/concept/making-the-grade/.docs/hints.md

```diff
@@ -5,11 +5,12 @@
 - `while` loops are used for _indefinite_ (uncounted) iteration
 - `for` loops are used for _definite_, (counted) iteration.
 - The keywords `break` and `continue` help customize loop behavior.
+- `std::to_string()` from `string` library can be used to convert a integer to string
 
 Also being familiar with the following can help with completing the tasks:
 
 - [`vectors`][vectors]: indexing, size, [`<vector>.emplace_back`][emplace], [`<vector>.pop_back()`][pop].
-- [`string`][string]: using the `+` to concatenate strings.
+- [`string`][string]: using the `+` to concatenate strings, integer to string conversion, [`to_string(int)`][to_string].
```

Comment:
Do you have the link to `to_string`as well?

Reply:
Sorry.. can i link a external website like cppreference or something ?

Reply:
Yes, you need to add the actual link somewhere. See the links section in the [markdown guide](https://exercism.org/docs/building/markdown/markdown).

Reply:
Done...